package.json

```diff
@@ -51,27 +56,28 @@
     "@messageformat/core": "^3.0.1",
     "@ngrx/store": "10.1.2",
     "@ngx-formly/core": "5.6.1",
-    "@stackblitz/sdk": "1.3.0",
+    "@stackblitz/sdk": "^1.3.0",
     "@types/yargs": "17.0.5",
     "@vcd/angular-client": "0.0.2-alpha.1",
     "@vcd/bindings": "9.1.1",
-    "@vmw/ng-live-docs": "0.0.15",
     "@webcomponents/custom-elements": "^1.5.0",
     "@webcomponents/webcomponentsjs": "2.0.0",
     "@wessberg/ts-evaluator": "0.0.25",
-    "angular-cli-ghpages": "0.6.2",
+    "angular-cli-ghpages": "1.0.0",
     "classlist.js": "1.1.20150312",
     "codecov.io": "0.1.6",
     "copyfiles": "^2.4.1",
+    "cross-spawn": "^7.0.3",
     "karma-coverage": "~2.0.3",
     "mousetrap": "1.6.5",
     "postcss": "^8.4.4",
-    "prismjs": "1.25.0",
+    "prismjs": "^1.23.0",
     "properties": "1.2.1",
-    "rbradford-compodoc": "1.1.11",
+    "rbradford-compodoc": "^1.1.11",
     "rxjs": "6.5.5",
+    "schematics-utilities": "^2.0.2",
     "systemjs": "0.20.19",
-    "tsickle": "0.39.1",
+    "tsickle": "^0.39.1",
```

Comment:
NIT: exact versions?

Reply:
Gave up using exact version, they caused too much trouble when two packages depended on the same version... The real benefit was so it wouldn't be changed when something else is installed. However, doing that may force you to have multiple version of packages causing bundle blot